eventually/authentication/models.py

```diff
@@ -60,10 +88,14 @@ def delete_by_id(user_id):
             user.delete()
             return True
         except CustomUser.DoesNotExist:
-            pass
+            return -1
 
     @staticmethod
-    def create(first_name, middle_name, last_name, email, password):
+    def create(first_name='first-name',
+               middle_name='middle-name',
+               last_name='last-name',
+               email='email@smth.com',
+               password='123456'):
         """
         :param first_name: first name of a user
```

Comment:
indicate the type of the function parameter with sphinx.

Reply:
Done.